nservicebus/messaging/databus/azure-blob-storage_config_ABSDataBus_[4.0,5.0).partial.md

```diff
@@ -0,0 +1,25 @@
+### Configuring the BlobServiceClient
+
+There are several ways to configure the `BlobServiceClient`. 
+
+#### Using a preconfigured BlobServiceClient
+
+A fully configured `BlobServiceClient` can be set through the settings:
+
+snippet: AzureDataBusConfigureServiceClient
+
+#### Using a custom provider
+
+A custom provider can be declared that provides a fully configured `BlobServiceClient`:
+
+snippet: CustomBlobServiceClientProvider
+
+The provider is then registered in the container:
```

Comment:
Are we risking that container is ambiguous because in relation to blobs container has a different meaning that DI container?